bin/test-workflow/6_app_deploy_backend.sh

```diff
@@ -52,5 +53,15 @@ helm install "$app_name" bitnami/postgresql -n "$TEST_APP_NAMESPACE_NAME" --debu
     --set securityContext.fsGroup="999" \
     --set postgresqlDatabase="test_app" \
     --set postgresqlUsername="test_app" \
-    --set postgresqlPassword="$SAMPLE_APP_BACKEND_DB_PASSWORD"
+    --set postgresqlPassword="$SAMPLE_APP_BACKEND_DB_PASSWORD")
+
+if [[ "$PLATFORM" == "openshift" ]]; then
+  args+=(--set containerSecurityContext.enabled=false \
```

Comment:
Nice work figuring out how to get the persistence working on OpenShift!!!

It might help other readers to add a comment pointing to the Bitnami doc for this (just above the conditional two lines up???):
https://github.com/bitnami/charts/tree/master/bitnami/postgresql#differences-between-bitnami-postgresql-image-and-docker-official-image